lib/interferon.rb

```diff
@@ -65,10 +65,10 @@ def read_alerts
       abort("no such directory #{path} for reading alert files") \
         unless Dir.exists?(path)
 
-      Dir.glob(File.join(path, '*.rb')) do |alert_file|
+      Dir.glob(File.join(path, '**/*.rb')) do |alert_file|
         break if @request_shutdown
         begin
-          alert = Alert.new(alert_file)
+          alert = Alert.new(@alerts_repo_path, alert_file)
```

Comment:
ok, in that case i guess you can pass `path` instead of `@alerts_repo_path` to the constructor. then, you'll just have to strip `path` from the beginning of `alert_file`